lightning/src/ln/channelmanager.rs

```diff
@@ -4288,6 +4306,7 @@ impl<Signer: Sign, M: Deref , T: Deref , K: Deref , F: Deref , L: Deref >
 							if let Some(short_id) = chan.get_short_channel_id() {
 								short_to_id.remove(&short_id);
 							}
+							self.pending_events.lock().unwrap().push(events::Event::ChannelClosed { channel_id: chan.channel_id(),  err: ClosureDescriptor::ProcessingError });
```

Comment:
I dunno if I'd call this ProcessingError, its really "peer disconnected before channel was initialized".